src/og.tsx

```diff
@@ -116,3 +115,365 @@ export default app.on('GET', ['/airdrop/', '/airdrop/:handle?'], async (c) => {
+		artistTier ? loadImage(c, `/icons/Token${artistTier}.svg`) : null,
+		commenterTier ? loadImage(c, `/icons/Token${commenterTier}.svg`) : null,
+	]);
+
+	return (
+		<div
+			style={{
+				display: 'flex',
+				flexDirection: 'column',
+				height: '100%',
+				width: '100%',
+				backgroundColor: '#FFF',
+			}}
+		>
+			<div
+				style={{
+					display: 'flex',
+					gap: '40px',
+					padding: '40px',
+					justifyContent: 'space-between',
+					background: 'linear-gradient(-22deg, #5b23e1 0%, #a22feb 100%)',
+					boxSizing: 'border-box',
+				}}
+			>
+				<div
+					style={{
+						display: 'flex',
+						flex: '1 1 0',
+						flexDirection: 'row',
+						gap: '24px',
+						alignItems: 'center',
+					}}
+				>
+					{trackArtwork ? (
+						<img
+							src={trackArtwork}
+							alt="Track Artwork"
+							height={120}
+							width={120}
+							style={{ width: '120px', height: '120px', borderRadius: '10px' }}
+						/>
+					) : null}
+					<div
+						style={{
+							display: 'flex',
+							flex: '1 1 0',
+							flexDirection: 'column',
+							gap: '4px',
+							height: '120px',
+							paddingTop: '4px',
+							alignSelf: 'center',
+						}}
+					>
+						<h2
+							style={{
+								margin: 0,
+								color: '#FFF',
+								fontSize: '48px',
+								fontWeight: '800',
+								lineHeight: '54px',
+								overflow: 'hidden',
+								textOverflow: 'ellipsis',
+								whiteSpace: 'nowrap',
+							}}
+						>
+							{trackName}
+						</h2>
+
+						<div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', gap: '8px' }}>
+							<p
+								style={{
+									display: 'flex',
+									alignItems: 'center',
+									gap: '8px',
+									margin: 0,
+									color: '#FFF',
+									fontSize: '40px',
+									fontWeight: '300',
+									lineHeight: '48px',
+									overflow: 'hidden',
+									textOverflow: 'ellipsis',
+									whiteSpace: 'nowrap',
+								}}
+							>
+								By {artistName}
+							</p>
+							<div style={{ display: 'flex', alignItems: 'center', gap: '8px' }}>
+								{isArtistVerified && verifiedIconWhite ? (
+									<img
+										src={verifiedIconWhite}
+										alt="Verified"
+										height={32}
+										width={32}
+										style={{ width: '32px', height: '32px', flexBasis: '32px' }}
+									/>
+								) : null}
+								{artistTier && artistTierIcon ? (
+									<img
+										src={artistTierIcon}
+										alt="Artist Tier"
+										height={32}
+										width={32}
+										style={{ width: '32px', height: '32px', flexBasis: '32px' }}
+									/>
+								) : null}
+							</div>
+						</div>
+					</div>
+				</div>
+				<div style={{ display: 'flex', alignSelf: 'flex-start', width: '200px' }}>
+					{audiusLogo ? (
+						<img
+							src={audiusLogo}
+							alt="Audius Logo"
+							height={48}
+							width={200}
+							style={{ width: '200px' }}
+						/>
+					) : null}
+				</div>
+			</div>
+			<div
```

Comment:
this indentation is wild lol

Reply:
yea it looks worse in github bc of the tabs instead of spaces